hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/upgrade/BasicUpgradeFinalizer.java

```diff
@@ -218,14 +218,15 @@ private void persistStorage(Storage config) throws IOException {
 
   protected void emitNOOPMsg(String feature) {
     String msg = "No finalization work defined for feature: " + feature + ".";
-    String msg2 = "Skipping.";
 
     logAndEmit(msg);
-    logAndEmit(msg2);
   }
 
   protected void emitStartingMsg() {
-    String msg = "Finalization started.";
+    String msg = "Finalization started.\n";
+    msg += "Existing pipelines and containers will be closed during Upgrade.\n";
```

Comment:
If we add here, then OM finalization will also print out the message. Can we print out the message after the actual work is done (freeze pipeline creation, pipelines are closed etc)?

Reply:
yup, I thought I was making change in ScmFinalizer. Fixed now.